src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Endpoint.cs

```diff
@@ -0,0 +1,211 @@
+            """;
+
+        return await dbConnection.QuerySingleOrDefaultAsync<ParentResource>(new CommandDefinition(
+            query,
+            new
+            {
+                code
+            },
+            cancellationToken: ct));
+    }
+
+    private sealed record ParentResource(
+        int Id,
+        string Code,
+        string DisplayName,
+        string ShortName,
+        ResourceType ResourceType,
+        string LicenseInfo);
+
+    private static async Task<IReadOnlyList<ParentResourceLocalizationWithResourceContentCount>>
+        GetParentResourceLocalizationWithResourceContentCountAsync(
+            DbConnection dbConnection,
+            int parentResourceId,
+            IReadOnlyList<int>? languageIds,
+            CancellationToken ct)
+    {
+        var parameters = new DynamicParameters(new { parentResourceId });
+
+        string? parentResourceLocalizationsWhereClause = null;
+        string? resourceContentCountWhereClause = null;
+        if (languageIds is not null or [])
+        {
+            parameters.Add(nameof(languageIds), languageIds);
+            parentResourceLocalizationsWhereClause = """
+                WHERE
+                    x.LanguageId IN @languageIds
+                """;
+            resourceContentCountWhereClause = "rc.LanguageId IN @languageIds AND";
+        }
+
+        // there is no English data in the ParentResourceLocalizations table so fetch it from the main ParentResources table
+        var query = $"""
+            SELECT
+                x.LanguageId,
+                x.DisplayName
+            FROM
+            (
+                SELECT
+                    1 AS LanguageId,
+                    pr.DisplayName
+                FROM
+                    ParentResources pr
+                WHERE
+                    pr.Id = @parentResourceId
+
+                UNION
+
+                SELECT
+                    prl.LanguageId,
+                    prl.DisplayName
+                FROM
+                    ParentResourceLocalizations prl
+                WHERE
+                    prl.ParentResourceId = @parentResourceId
+            ) x
+            {parentResourceLocalizationsWhereClause}
+            ORDER BY
+                x.LanguageId;
+
+            SELECT
+                rc.LanguageId,
+                COUNT(rcv.Id) AS ResourceCount
+            FROM Resources r
+                JOIN ResourceContents rc ON rc.ResourceId = r.Id
+                JOIN ResourceContentVersions rcv ON rcv.ResourceContentId = rc.Id
+            WHERE
+                r.ParentResourceId = @parentResourceId AND
+                {resourceContentCountWhereClause}
+                rcv.IsPublished = 1
+            GROUP BY
+                rc.LanguageId;
+            """;
+
+        // querying separately and joining in memory saves significant time over trying to join in the DB
+        var reader = await dbConnection.QueryMultipleAsync(new CommandDefinition(
```

Comment:
Dapper allows for multiple `SELECT`s in a single call to the DB which can sometimes help with performance to avoid doing a DB JOIN operation.  My method here is kind of like a manual `SplitQuery()` except EF makes two separate DB calls and this is only one.